data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
@wgrriffel not pending to any side, but comparing to Dota is too shallow here. TFS IS meant to be MUCH more flexible, as it does not accept any user custom script as Dota does (as you mentioned, I don't play it), so all scripts are to be validated by the server owner.

That said, I'd like more of the core moved to Lua even though it's easily "crashable" by a bad scripter. Store inbox being a Lua event would be easy to e.g. notify a player if something happens, or add more/remove some checks. To me, it's a very valid tradeoff.

You guys should open a new issue to discuss that in a more generic way, this review is getting way too large and hard to follow.